frontend/src/pages/RequestPasswordResetPage.js

```diff
@@ -40,54 +39,44 @@ export default function RequestPasswordResetPage({ user }) {
 	}
 	return (
 		<CenterFlexBox>
-			<Avatar sx={{ m: 1, bgcolor: "secondary.main" }}>
-				<LockOutlinedIcon />
-			</Avatar>
-			<Typography component="h1" variant="h5">
+			<ParagraphTypography component="h1" variant="h5" style={{ paddingBottom: "29px" }}>
 				Request Password Reset
-			</Typography>
+			</ParagraphTypography>
 			{resetWasRequested ?
 				<Box sx={{ m: 4 }} >
-					<Alert severity="info">
-						You will receive an email shortly explaining how you can reset your password.
-					</Alert>
+					<ParagraphTypography>You will receive an email shortly explaining how you can reset your password.</ParagraphTypography>
 				</Box>
 				:
 				<Box component="form" onSubmit={handleSubmit} noValidate sx={{ mt: 1 }}>
-					<Typography>Enter your email and click the button below to request a password reset</Typography>
+					<ParagraphTypography>Enter your email and click the button below to request a password reset:</ParagraphTypography>
 					<TextField
+						sx = {{ ...inputFieldStyle, marginBottom: 0 }}
 						margin="normal"
 						required
 						fullWidth
 						id="email"
-						label="Email Address"
+						label="Email"
 						name="email"
 						autoComplete="email"
 						autoFocus
 					/>
-					<>
-						{errorMessage !== "" && <Grid item xs={12}><Alert severity="error">{errorMessage}</Alert></Grid>}
-						<Button
+					{errorMessage !== "" && <Grid item xs={12} style={{ paddingTop: "18px" }}><Alert severity="error">{errorMessage}</Alert></Grid>}
+					<div style={{ overflowX: "none", display: "flex", paddingTop: "18px", paddingBottom: "18.5px" }}>
+						<RoundedEdgesButton
 							type="submit"
 							fullWidth
 							variant="contained"
-							sx={{ mt: 3, mb: 2 }}>
+							style={{ backgroundColor: "#008425", width: "250px", marginLeft: "auto" }} >
```

Comment:
This is a welcome change. It's good to prefer good old CSS over cryptic abbreviations that need to be looked up by people who don't know MUI.